ts/navigation/params/MainTabParamsList.ts

```diff
@@ -5,7 +5,7 @@ import { SERVICES_ROUTES } from "../../features/services/common/navigation/route
 
 export type MainTabParamsList = {
   [MESSAGES_ROUTES.MESSAGES_HOME]: undefined;
-  [ROUTES.WALLET_HOME]: WalletHomeNavigationParams;
+  [ROUTES.WALLET_HOME]: WalletHomeNavigationParams | undefined;
```

Comment:
Better to revert this and use the correct navigation where needed. What do you think?

```ts
navigate(ROUTES.MAIN, {
  screen: ROUTES.WALLET_HOME,
  params: { newMethodAdded: false }
});
```

Reply:
I agree! Thank you!